client/layout/404.jsx

```diff
@@ -0,0 +1,46 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import ReactDom from 'react-dom';
+import page from 'page';
+
+/**
+ * Internal dependencies
+ */
+import EmptyContent from 'components/empty-content';
+import { setSection } from 'state/ui/actions';
+
+let Page404 = React.createClass( {
+	displayName: 'Page404',
```

Comment:
`displayName` is obsolete when React component is assigned to a variable. It just duplicates the same name.